library/sources/http-server/checkIfRequestIsBlocked.ts

```diff
@@ -65,38 +65,47 @@ export function checkIfRequestIsBlocked(
     return true;
   }
 
-  const result = context.remoteAddress
+  const blockedIPs = context.remoteAddress
     ? agent.getConfig().isIPAddressBlocked(context.remoteAddress)
```

Comment:
A bit confusing that the method `isIPAddressBlocked` can also return a non-empty array if the ip is not blocked (monitor mode). Same for user agents. Maybe rename method?